nilearn/plotting/img_plotting.py

```diff
@@ -1463,3 +1463,128 @@ def plot_connectome_strength(adjacency_matrix, node_coords, node_size="auto",
         display = None
 
     return display
+
+
+def plot_carpet(img, mask_img=None, detrend=True, output_file=None,
+                figure=None, axes=None, title=None):
+    """Plot an image representation of voxel intensities across time.
+
+    This figure is also known as the "carpet plot" or "Power plot".
+
+    Parameters
+    ----------
+    img : Niimg-like object
```

Comment:
Could you write the docstrings as in #2450 ?

Reply:
I think it should be good now, unless I missed something in #2450.

Reply:
thx.